spec/simulator_app_life_cycle_spec.rb

```diff
@@ -46,6 +46,16 @@ def self.simctl_thinks_app_is_installed?(device, app)
       expect(core_sim.app_is_installed?).to be_truthy
     end
 
+    it "installs app on simulator indicated with alias by --device-id" do
```

Comment:
Let's skip this test for now because it requires that we create a named simulator with `simctl`.  This is not really a use case.

Replace what you have with:

```
it "installs app on simulator indicated with alias by --device-id"
```

Without the `do / end` body, this test will be "pending".